README.md

```diff
@@ -64,7 +69,16 @@ argument. Finally, demonstrate calling `doSomething` with a function.**
 
 Your Answer:
 ```js
-// write code here
+var doSomething = function (thingToDo) {
+  thingToDo = $(.listItem[0]).html();
```

Comment:
We were looking for an answer without reference to the DOM. Let me know if you have any questions about that.